S3Proxy/options.go

```diff
@@ -10,4 +13,12 @@ var Options = options{}
 func LoadDefaultOptions() {
 	Options.CacheDir = "/tmp/S3Proxy/"
 	Options.BindAddress = ":9090"
+	Options.BucketCacheTTL = time.Duration(1 * time.Hour)
+}
+
+// Options to be loaded in for unit tests
+func LoadTestOptions() {
```

Comment:
Rather use LoadDefaultOptions and override the Options.BucketCacheTTL in the test file.